pymc/aesaraf.py

```diff
@@ -1015,3 +1019,144 @@ def compile_pymc(
         **kwargs,
     )
     return aesara_function
+
+
+def compile_forward_sampling_function(
```

Comment:
I think this should be in `sampling.py`

Reply:
Why?

Reply:
It's implementing a specific sampling algorithm, it's not just a Aesara helper